bool success
float32 x
float32 y
float32 z
float32[] vector
geometry_msgs/Pose[] pose_camera_aruco
int32 id_aruco
bool[] aruco_found
